pages/data-design/avo-tracking-plan/journeys.mdx

```diff
@@ -0,0 +1,233 @@
+import { Callout } from 'nextra/components'
+
+# Journeys (alpha)
+
+Use Journeys to visually design analytics for a product update by composing a flow from product images or screenshots that represent each step of your experience. Add triggers for each step and connect them to events and variants in your tracking plan. If you opt in to Avo Intelligence for Journeys, AI can generate triggers from images and suggest matching existing events/variants to speed up data design.
```

Comment:
I recall @klaraavo suggesting we stop referring to variants specifically everywhere in our docs, they fall under the events umbrella:

```suggestion
Use Journeys to visually design analytics for a product update by composing a flow from product images or screenshots that represent each step of your experience. Add triggers for each step and connect them to events in your tracking plan. If you opt in to Avo Intelligence for Journeys, AI can generate triggers from images and suggest matching existing eventsto speed up data design.
```